grid-compose/cmd/version.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var version = "v0.0.1"
```

Comment:
check how to properly do the versioning https://github.com/threefoldtech/tfgrid-sdk-go/blob/0c4a82b2e3fe68c8b70d4fac89bb3e31fecf8566/grid-cli/cmd/version.go